docs/core-concepts/introduction.md

```diff
@@ -1,41 +1,55 @@
 ---
 title: What is HAMi?
 slug: /
-
 ---
 
-## HAMi: Heterogeneous AI Computing Virtualization Middleware
+## TOC {#toc}
+
+- [HAMi: Heterogeneous AI Computing Virtualization Middleware](#hami-heterogeneous-ai-computing-virtualization-middleware)
+- [Why HAMi](#why-hami)
+- [What's Next](#whats-next)
+
+## HAMi: Heterogeneous AI Computing Virtualization Middleware {#hami-heterogeneous-ai-computing-virtualization-middleware}
+
+Heterogeneous AI Computing Virtualization Middleware (HAMi), formerly known as
+k8s-vGPU-scheduler is an "All-in-One" chart designed to manage Heterogeneous
+AI Computing Devices in a k8s cluster. It can provide the ability to share
+Heterogeneous AI devices among tasks.
+
+HAMi is a [Cloud Native Computing Foundation](https://cncf.io/) SandBox project
+  & [Landscape project](https://landscape.cncf.io/?item=orchestration-management--scheduling-orchestration--hami)
+  & [CNAI Landscape project](https://landscape.cncf.io/?group=cnai&item=cnai--general-orchestration--hami).
+
+## Why HAMi {#why-hami}
+
+- **Device sharing**
 
-Heterogeneous AI Computing Virtualization Middleware (HAMi), formerly known as k8s-vGPU-scheduler, is an "all-in-one" chart designed to manage Heterogeneous AI Computing Devices in a k8s cluster. It can provide the ability to share Heterogeneous AI devices among tasks.
+  - Support multiple Heterogeneous AI Computing devices
+  - Support device-sharing for multi-device containers
 
-HAMi is a [Cloud Native Computing Foundation](https://cncf.io/) sandbox project & [Landscape project](https://landscape.cncf.io/?item=orchestration-management--scheduling-orchestration--hami) & [CNAI Landscape project](https://landscape.cncf.io/?group=cnai&item=cnai--general-orchestration--hami).
+- **Device Memory Control**
 
-## Why HAMi:
-- __Device sharing__
-    - Support multiple Heterogeneous AI Computing devices
-    - Support device-sharing for multi-device containers
+  - Hard limit inside container
```

Comment:
```suggestion
    - Hard limit inside container
```
The indentation of 2-spaces does not work. MkDocs requires to indent 4-spaces by default.